tests/test_cli_compile.py

```diff
@@ -11,6 +11,8 @@
 
 from .constants import MINIMAL_WHEELS_PATH, PACKAGES_PATH
 
+is_pypy = "__pypy__" in sys.builtin_module_names
```

Comment:
I'm not sure what's the canonical way to detect python implementation, maybe [platform.python_implementation() == "PyPy"](https://docs.python.org/3/library/platform.html#platform.python_implementation)?

Reply:
That may work too.

Reply:
Okay, feel free to ignore the suggestion 👍🏻